### DISCUSSION TITLE
Is there a way to use Memoria with the Windows Store version of FFIX?

### DISCUSSION THREAD

**Discussion by gwynlordofcinder** [Albeoris/Memoria#359] [Q&A] [answered]
I have the Windows Store version and I'm wondering if there is any way to install mods for it.

**Comment by snouz** [comment, marked as answer]
Last time I checked (that was 4y ago so that might have changed), the Windows Store did not support modding at all (or in some very limited cases), so that means nobody here has tries making the mod(s) work with it. Maybe you have more info than me though?

**Comment by Albeoris** [comment]
> I have the Windows Store version and I'm wondering if there is any way to install mods for it.

@gwynlordofcinder 
I guess, you need to do some research starting from where the game files? :) 
You can try to ask ChatGPT, Google or look for "FF9.exe" on the disk.
Also you can try to use PowerShell [cmdlet](https://learn.microsoft.com/en-us/powershell/module/appx/get-appxpackage?view=windowsserver2022-ps):
```powershell
get-appxpackage -name *Fantasy*
```
And check the `InstallLocation` parameter.

I guess it must be somewhere here: `C:\Program Files\WindowsApps`

Than just copy Memoria.Patcher.exe to this folder and try to run it `as Adminitstrator`.
We are waiting for result of your experiment. :)
